src/ScottPlot/GlobalFont.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Text;
+
+namespace ScottPlot
+{
+    public static class GlobalFont
+    {
+        /// <summary>
+        ///     This method will check for several preffered fonts
+        ///     to exist on the system and return the exact name.
+        /// </summary>
+        /// <returns></returns>
+        public static string GetDefault()
+        {
+            foreach (FontFamily font in FontFamily.Families)
```

Comment:
In your implementation you always get `DEJAVU` font if `FontFamily.Families` in alphabetical order.
To make ordered font select, like at first `SEGOE`, at second `DEJAVU`, at third `SANS` you can use linq like below.
Looks not so clear, but always select `SEGOE` if available:
```c#
string[] PrefFonts = new string[]{"SEGOE", "DEJAVU", "SANS"};
var PrefSystemFont = PrefFonts
    .SelectMany( pf => FontsFamily.Families, (PrefFont, SystemFont) => new { PrefFont, SystemFont })
    .Where( pfsf => pfsf.SystemFont.Name.ToUpper().Contains(pfsf.PrefFont.Name))
    .Select(x => x.SystemFont.Name);   
if (PrefSystemFont.Any())
    return PrefSystem.First();
```